### DISCUSSION TITLE
使用2FA后,无法通过alist挂载的方式添加别的list

### DISCUSSION THREAD

**Discussion by wontdoanything** [AlistGo/alist#5116] [Q&A] [answered]
服务端版本号:3.26.0
客户端版本号:3.26.0
条件:服务端开启2FA
状况:客户端添加存储,采用alist V3的方式,客户端提示
failed init storage: request failed,code: 402, message: Invalid 2FA code
问题:是目前不支持2FA方式加载alist吗,目前在文档和讨论中都未找到说明，还请懂的人帮忙解答

**Comment by xhofe** [comment, marked as answer]
https://alist.nn.ci/config/other.html#token

**Reply by wontdoanything** [reply]
原来如此,把认证用的token填入到令牌里就好了.
之前我是直接创建了另一个不带2FA的账号去加载alist的,避开了这个问题
